parachain/pallets/verifier-lightclient/src/mock.rs

```diff
@@ -1,6 +1,8 @@
 // Mock runtime
+#![cfg(test)]
```

Comment:
Any reason you added this here? It shouldn't be necessary in this context, since `mock.rs` is already marked to be conditionally compiled due to https://github.com/Snowfork/polkadot-ethereum/blob/eth-header-verification-3/parachain/pallets/verifier-lightclient/src/lib.rs#L20

Reply:
Ha, I didn't actually notice the imports and thought we were compiling these in the regular build. Good to know we're not :D